src/components/structures/RoomStatusBar.js

```diff
@@ -24,7 +24,11 @@ import WhoIsTyping from '../../WhoIsTyping';
 import MatrixClientPeg from '../../MatrixClientPeg';
 import MemberAvatar from '../views/avatars/MemberAvatar';
 import Resend from '../../Resend';
-import { showUnknownDeviceDialogForMessages } from '../../cryptodevices';
+import {
```

Comment:
should we `import * as cryptodevices from '../../cryptodevices'` rather than having a long list?

Reply:
done